auth.go

```diff
@@ -4,99 +4,119 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"errors"
-	log "github.com/sirupsen/logrus"
-	"go.mozilla.org/hawk"
 	"io"
 	"io/ioutil"
 	"mime"
 	"net/http"
 	"strings"
 	"time"
+
+	log "github.com/sirupsen/logrus"
+	"go.mozilla.org/hawk"
```

Comment:
Any reason for this change? Is this something `gofmt` does now?

Reply:
yes 😄

Reply:
👍